### PR TITLE
Loop detection in a Linked List using Python

### PR DESCRIPTION
**Pull request by yashmita**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Added code for detecting loop in a linked list using Python.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Issue #5115

### CODE REVIEW

**Comment by yashmita** [comment]
In Deepsouce: Python, it's showing "Indentation has tabs when only spaces are expected. Change all tabs to spaces."
I tried correcting the indentation, but I'm still unable to pass all the checks.
And the code is running perfectly fine on the online IDE.
Kindly suggest what to do

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@yashmita You can format your code here http://pep8online.com.

**Comment by yashmita** [comment]
> @yashmita You can format your code here http://pep8online.com.

Thank you! Now it has passed all the checks.
Kindly review it.

**Review by iamrajiv — APPROVED** [review]

File: Python/ds/loop_detection_in_LL.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+# 2.
+# n= 4
+# 6 8 2 1
+# pos = -1
+# Output:
+# Loop not found!
+
+# TIME COMPLEXITY: O(n), to traverse the linked list
+# where 'n' denotes the length of the linked list.
+# SPACE COMPLEXITY: O(1), no extra space used.
```
Comment (ankitaggarwal23) [inline Python/ds/loop_detection_in_LL.py]: Use multiline comments. Write the outputs in a way your code is producing also remove extra test case

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

File: Python/ds/loop_detection_in_LL.py (hunk 1 above)
Comment (yashmita) [reply Python/ds/loop_detection_in_LL.py]: > Use multiline comments. Write the outputs in a way your code is producing also remove extra test case

I used multiline comments earlier, but in Deepsource: Python it gives an error of Anti-pattern. So, I used single-line comments.
But, now I've changed them back to multiline and I've also updated other changes.
Kindly have a look.

Comment (yashmita) [reply Python/ds/loop_detection_in_LL.py]: @ankitaggarwal23  I used multiline comments earlier, but in Deepsource: Python it gives an error of Anti-pattern. So, I used single-line comments.
But, now I've changed them back to multiline and I've also updated other changes.
Kindly have a look.

**Review by ankitaggarwal23 — APPROVED** [review]

**Pull request merged**